src/puppeteer-tests/utils/Selectors.js

```diff
@@ -0,0 +1,113 @@
+
+// Selectors that are used across all platfroms are written directly in SelectorsMaster
+const CommonSelectors = {
+	logIn: '.login-link',
+	logInEmail: '#loginByEmail input[name=email]',
+	logInPassword: '#loginByEmail input[name=password]',
+	Save: '.SavePublishDiscard button[name=save]',
+	SaveAs: '.SavePublishDiscard button[name=save-as]',
+	Publish: '.SavePublishDiscard button[name=publish]',
+	DiscardEdits: '.SavePublishDiscard button[name=discard]',
+	Unpublish: '.SavePublishDiscard button[name=unpublish]',
+	Delete: '.SavePublishDiscard button[name=delete]',
+	facebookLogin: '.social-signin .facebook',
+	twitterLogin: '.social-signin .twitter',
+	Filter: '.form-control[name=f]'
+};
+
+const TestAs = {
+	FirstCharityIcon: "chooser-list.ready > a",
+	Banner: "#banner",
+	ClickToPlay: ".gl-play-video",
+	TwitterLink: "a[alt='Twitter']",
+	LinkedInLink: "a[alt='LinkedIn']",
+	FacebookLink: "a[alt='Facebook']"
+};
+
+const FacebookSelectors = {
+	username: '#email',
+	password: '#pass',
+	login: '#loginbutton input',
+	continue: '#u_0_4 > div._58xh._1flz > div._1fl- > div._2mgi._4k6n > button'
+};
+
+const TwitterSelectors = {
+	apiUsername: '#username_or_email',
+	apiPassword: '#password',
+	apiLogin: '#allow',
+	username: 'fieldset > div:nth-child(2) > input',
+	password: 'div:nth-child(3) > input',
+	login: 'div.clearfix > button'
+};
+
+// Portal selectors
+const CRUD = {
+	Save: `div.SavePublishDiscard [name=save]`,
+	Publish: `div.SavePublishDiscard [name=publish]`,
+	Discard: `div.SavePublishDiscard [name=discard]`,
+	Delete: `div.SavePublishDiscard [name=delete]`
+};
+
+const ListLoad = {
+	Create: `.ListLoad .btn-create`
+};
+
+const Environment = {
+	logIn: '#top-right-menu .login-link a'
+};
+
+const Main = {
+	FirstCharityIcon: ".chooser-list .ch1",
+	Banner: "div.glhdr"
+};
+
+const Advertiser = {
```

Comment:
Should this be in the PortalSelectors file?

Reply:
Possibly - these files are copied from the portal tests so I left them as is up till now. Can move it if thats more suitable